docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
+
+const TESTNET_NODE_URL = 'https://testnet.aeternity.io'
+const MAINNET_NODE_URL = 'https://mainnet.aeternity.io'
+const COMPILER_URL = 'https://compiler.aepps.com';
 
+export default {
+  data: () => ({
+    aeSdk: null,
+    publicAddress: '',
+    balance: 0,
+    node: null,
+    accounts: null
+  }),
+}
+```
+
+## 2. Initialize the `RpcAepp` Stamp
+
+```js
 async created () {
   this.aeSdk = await RpcAepp({
     name: 'Simple Aepp',
     nodes: [
-      { name: 'ae_uat', instance: await Node({ url: TEST_NET_NODE_URL }) },
-      { name: 'ae_mainnet', instance: await Node({ url: MAIN_NET_NODE_INTERNAL_URL }) }
+      { name: 'ae_uat', instance: await Node({ url: TESTNET_NODE_URL }) },
+      { name: 'ae_mainnet', instance: await Node({ url: MAINNET_NODE_URL }) }
     ],
     compilerUrl: COMPILER_URL,
-    onNetworkChange: async (params) => {
-      this.aeSdk.selectNode(params.networkId)
-      this.nodeInfoResponse = await this.aeSdk.getNodeInfo()
+    onNetworkChange: async ({ networkId }) => {
+      this.aeSdk.selectNode(networkId)
+      this.node = await this.aeSdk.getNodeInfo()
     },
-    onAddressChange:  async (addresses) => {
-      this.pub = await this.aeSdk.address()
-      this.balance = await this.aeSdk.balance(this.pub).catch(e => '0')
-      this.addressResponse = await this.aeSdk.address()
+    onAddressChange: async () => {
+      this.publicAddress = await this.aeSdk.address()
+      this.balance = await this.aeSdk.balance(this.publicAddress)
     },
     onDisconnect: () => {
-      this.resetState()
+      // you may want to reset state here
       alert('Disconnected')
     }
   })
-  this.height = await this.aeSdk.height()
-  // start looking for wallets, see step 2.
   this.scanForWallets()
-  // open iframe with Wallet if run in top window
-  if (window === window.parent) this.openReverseIframe()
 }
 ```
 
-## 2. Scan for wallets
+## 3. Scan for wallets
 ```js
-scanForWallets () {
-  const handleWallets = async function ({ wallets, newWallet }) {
-    newWallet = newWallet || Object.values(wallets)[0]
-    if (confirm(`Do you want to connect to wallet ${newWallet.name}`)) {
-      this.detector.stopScan()
-      // connect to the wallet, see step 3.
-      await this.connectToWallet(newWallet)
+methods: {
+  async scanForWallets()  {
+    const handleWallets = async function ({ wallets, newWallet }) {
+      newWallet = newWallet || Object.values(wallets)[0]
+      if (confirm(`Do you want to connect to wallet ${newWallet.name}`)) {
+        detector.stopScan()
+        // connect to the wallet, see step 4.
+        await this.connectToWallet(newWallet)
+      }
     }
-  }
 
-  const scannerConnection = BrowserWindowMessageConnection({
-    connectionInfo: { id: 'spy' }
-  })
-  this.detector = WalletDetector({ connection: scannerConnection })
-  this.detector.scan(handleWallets.bind(this))
+    const scannerConnection = BrowserWindowMessageConnection({
+      connectionInfo: { id: 'spy' }
```

Comment:
Here would need some advise.
I would like to put comment in here but not really sure what this scanner is for. Maybe we can come up with better naming instead of comment?

Reply:
`scannerConnection` is a stamp instance that polls an aepp with wallet info object until gets a reply 🤷‍♀️  I have a plan to simplify this process in future

Reply:
Ok, thanks for the explanation. I leave it without comment for now